### PR TITLE
IFB

### PR DESCRIPTION
**Pull request by lecorguille**
Better late than never!

I just didn't manage to get IFB in the Member Sites list.

Ping @abretaud and @julozi to review before merge

### CODE REVIEW

**Comment by bgruening** [comment]
Uh, nice! Thanks @lecorguille ... let me know when this is ready! :)

**Review by abretaud — COMMENTED** [review]
Cool to add ifb! A few comments...

File: _includes/home_carousel_ifb_galaxycat.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="item carousel-item jumbotron">
+  <h2 class="display-3">Ask the GalaxyCat</h2>
+  <div class="col-md-6">
+  <p>Can't find a tool on usegalaxy.fr?<br/>
+  Why not search on the IFB Galaxy Catalog ?</p>
+  <button type="button" class="btn btn-primary btn-lg" onclick="window.open('http://galaxycat.france-bioinformatique.fr/')">Go to the ProteoRE thematic</button>
```
[the remark this answers is not in the document]
Comment (lecorguille) [reply _includes/home_carousel_ifb_galaxycat.html]: Fix

File: _config.yml [hunk 2]
```diff
@@ -145,8 +145,18 @@ team_sites:
 
         The development of Laniakea continues in the framework of [EOSC-Pillar](https://www.eosc-pillar.eu/) (Grant agreement ID: 857650) and [EOSC-Life](https://www.eosc-life.eu/) (Grant agreement ID: 824087) H2020 projects.
 
-        The [Laniakea@ReCaS](https://laniakea-elixir-it.github.io/laniakea_at_recas) service is provided with the support of [ELIXIR-ITALY](https://elixir-italy.org/) and [ReCaS-Bari](https://www.recas-bari.it). 
-
+        The [Laniakea@ReCaS](https://laniakea-elixir-it.github.io/laniakea_at_recas) service is provided with the support of [ELIXIR-ITALY](https://elixir-italy.org/) and [ReCaS-Bari](https://www.recas-bari.it).
+    ifb:
+      name: "Institut Français de Bioinformatique (IFB)"
```
[the remark this answers is not in the document]
Comment (lecorguille) [reply _config.yml]: Institut Français de Bioinformatique
or 
IFB
or
Elixir-FR ?

[the remark this answers is not in the document]
Comment (abretaud) [reply _config.yml]: I'd say IFB or ELIXIR-FR/IFB

[the remark this answers is not in the document]
Comment (lecorguille) [reply _config.yml]: Done

File: _bibliography/ifb.bib [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+---
+
+% ordered by year, then alphabetically
+
+# Publications
+
+# 2018
```
Comment (bgruening) [inline _bibliography/ifb.bib:9]: w4m publication?

Comment (lecorguille) [reply _bibliography/ifb.bib:9]: I thought that better none than few not well chosen.

If I add W4M, why not add proteore ... and oh, and you have forgot this one ... blablabla

We have a usegalaxy.fr meeting this week. I can add this point in the agenda.

Comment (bgruening) [reply _bibliography/ifb.bib:9]: fine with me ... do you think this is ready?

**Comment by julozi** [comment]
Hi guys, nothing to add for me ! This is nice !

File: _bibliography/ifb.bib (hunk 3 above)
Comment (lecorguille) [reply _bibliography/ifb.bib:9]: Ready 🚀

**Comment by bgruening** [comment]
Works for me, lets ship it. 

Thanks a lot, France!

**Pull request merged**